src/components/sds/Toast/index.tsx

```diff
@@ -0,0 +1,207 @@
+    isFilled?: boolean;
+  }) => getBackgroundColor(variant, isFilled)};
+  border: 1px solid ${PALETTE.dark.gray["06"]};
+  border-radius: ${px(8)};
+  padding: ${px(12)};
+  margin: ${px(8)} 0;
+  flex-direction: column;
+  gap: ${px(8)};
+  width: 100%;
+  shadow-color: #000;
+  shadow-offset: 0px 2px;
+  shadow-opacity: 0.25;
+  shadow-radius: 3.84px;
+  elevation: 5;
+`;
+
+const TitleContainer = styled.View`
+  flex-direction: row;
+  align-items: flex-start;
+  gap: ${px(8)};
+`;
+
+const IconContainer = styled.View`
+  width: ${px(16)};
+  height: ${px(16)};
+  margin-top: ${px(2)};
+  flex-shrink: 0;
+`;
+
+const TitleText = styled(Text)`
+  color: ${PALETTE.dark.gray["12"]};
+  font-size: ${px(14)};
+  line-height: ${px(20)};
+  font-weight: 600;
+`;
+
+const MessageText = styled(Text)`
+  color: ${PALETTE.dark.gray["11"]};
+  font-size: ${px(14)};
+  line-height: ${px(20)};
+  font-weight: 400;
+`;
+
+/**
+ * Toast component for displaying temporary notifications
+ *
+ * @example
+ * <Toast
+ *   variant="success"
+ *   title="Success!"
+ *   message="Your action was completed successfully."
+ *   duration={3000}
+ *   onDismiss={() => console.log('Toast dismissed')}
+ * />
+ */
+export const Toast: React.FC<ToastProps> = ({
+  variant,
+  title,
+  icon,
+  message,
+  duration = 3000,
+  onDismiss,
+  isFilled = false,
+}) => {
+  const fadeAnim = React.useRef(new Animated.Value(0)).current;
+  const slideAnim = React.useRef(new Animated.Value(-100)).current;
+
+  const dismissToast = useCallback(() => {
+    Animated.parallel([
+      Animated.timing(fadeAnim, {
+        toValue: 0,
+        duration: 300,
+        useNativeDriver: true,
+      }),
+      Animated.timing(slideAnim, {
+        toValue: -100,
+        duration: 300,
+        useNativeDriver: true,
+      }),
+    ]).start(() => {
+      onDismiss?.();
+    });
+  }, [fadeAnim, slideAnim, onDismiss]);
+
+  useEffect(() => {
+    // Animate in
+    Animated.parallel([
+      Animated.timing(fadeAnim, {
+        toValue: 1,
+        duration: 300,
+        useNativeDriver: true,
+      }),
+      Animated.timing(slideAnim, {
+        toValue: 0,
+        duration: 300,
+        useNativeDriver: true,
+      }),
+    ]).start();
+
+    // Auto dismiss after duration
+    const timer = setTimeout(dismissToast, duration);
+    return () => clearTimeout(timer);
+  }, [dismissToast, duration, fadeAnim, slideAnim]);
+
+  return (
+    <ToastWrapper>
+      <ToastContainer
+        variant={variant}
+        isFilled={isFilled}
+        style={{
+          opacity: fadeAnim,
+          transform: [{ translateY: slideAnim }],
+        }}
+      >
+        <TitleContainer>
+          <IconContainer>
+            {icon || (
+              <Icon.InfoCircle size={16} color={getIconColor(variant)} />
+            )}
+          </IconContainer>
+          <TitleText>{title}</TitleText>
+        </TitleContainer>
+        {message && <MessageText>{message}</MessageText>}
```

Comment:
```suggestion
        {message && <Text sm secondary>{message}</Text>}
```

Reply:
@danilosilvackl @Brunonascdev also for the text color you can check it in a few places like the below:
![Screenshot 2025-04-03 at 12 36 56](https://github.com/user-attachments/assets/b29ff44b-9d5f-4a7c-9f24-0cb155456963)